src/components/NewScreenshot.jsx

```diff
@@ -5,7 +5,7 @@ function NewScreenshot() {
   const [screenshotUrl, setScreenshotUrl] = useState("");
   const [image, setImage] = useState(null);
 
-  const handleSubmit = async (e) => {
+ const handleSubmit = async (e) => {
```

Comment:
Why removing the whitespace here? Let's keep the indentation as it should be :)

Reply:
I will consider the indentation for other commits. Maybe I moved it by accident.